pipwin/command.py

```diff
@@ -38,7 +38,7 @@ def _package_names(args):
         print("Provide a package name")
         sys.exit(0)
     else:
-        yield Requirement(args["<package>"])
+        yield Requirement(args["<package>"].lower())
```

Comment:
is this fixing any issue that we know about? i would rather keep the case if it's not affecting anything negatively

Reply:
fails for `pipwin install python-Levenshtein `